domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/charSequenceAssertions.kt

```diff
@@ -41,3 +42,6 @@ fun _isNotEmpty(subjectProvider: SubjectProvider<CharSequence>): Assertion =
 
 fun _isNotBlank(subjectProvider: SubjectProvider<CharSequence>): Assertion =
     AssertImpl.builder.createDescriptive(subjectProvider, IS_NOT, RawString.create(BLANK)) { it.isNotBlank() }
+
+fun <T : CharSequence> _mismatches(assertionContainer: Expect<T>, expected: Regex): Assertion =
+    AssertImpl.builder.createDescriptive(assertionContainer, MISMATCHES, expected) {!it.matches(expected)}
```

Comment:
Please use always ExpectImpl
```suggestion
    ExpectImpl.builder.createDescriptive(assertionContainer, MISMATCHES, expected) { !it.matches(expected) }
```